### DISCUSSION TITLE
Why does noisemadelling always report errors when importing. pbf data? I downloaded the data from geofabrik

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#574] [Q&A]
Why does noisemadelling always report errors when importing. pbf data? I downloaded the data from geofabrik
![image](https://user-images.githubusercontent.com/38131350/233895435-ae2c37a5-1e5a-4385-aa56-15b5217a504e.png)

**Comment by pierromond** [comment]
mmm I am not sure, but maybe it comes from the provider.

We are more used to use BBBIke provider. We should specify it in the documentation. Could you try and tell us ?
